language-server/protocol/extProtocol.ml

```diff
@@ -26,6 +26,7 @@ module Notification = struct
       type t = {
         textDocument : VersionedTextDocumentIdentifier.t;
         position : Position.t;
+        to_next_point : bool
       } [@@deriving yojson]
```

Comment:
> Look better to me. @rtetley would it be possible to make the boolean argument optional in the protocol so to be backwards compatible?

I believe it is enough to make ```to_next_point``` an option. i.e.:
```suggestion
      type t = {
        textDocument : VersionedTextDocumentIdentifier.t;
        position : Position.t;
        to_next_point : bool option;
      } [@@deriving yojson]
```

Reply:
But I guess there is a larger question here, and I feel kinda silly because I am the one that suggested making this part of the protocol instead of the server options. Are we going to use both versions of simultaneously ? If it makes sense to sometimes use the ```to_next_point: true``` version and false in the same document we should keep it that way. Otherwise we can revert to making it a server option. What do you think @gares ?

Reply:
I see no use case of using both behaviors at the same time :-/

Reply:
Well I think then we should revert to making it an option in the server that gets sent through the configuration request ! Sorry @Durbatuluk1701 😅

Reply:
All good, I think the iterations has made this an overall better solution for sure